internal/pkg/aws/ec2/ec2.go

```diff
@@ -39,9 +38,10 @@ type api interface {
 
 // Filter contains the name and values of a filter.
 type Filter struct {
-	// Name is the name of a filter that will be applied to subnets, for available filter names see: https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeSubnets.html
+	// Name is the name of a filter that will be applied to subnets,
```

Comment:
Maybe just "Name of a ..." and "Values of ...". Note that docstring usually starts with the func/var name.